src/lib/components/documents/Page.svelte

```diff
@@ -0,0 +1,47 @@
+<script lang="ts">
+  import type { ViewerMode } from "@/lib/api/types";
+
+  import { _ } from "svelte-i18n";
+
+  import { pageHashUrl } from "$lib/api/documents";
+
+  export let page_number: number;
+  export let mode: ViewerMode = "document";
+
+  $: id = pageHashUrl(page_number);
```

Comment:
`pageHashUrl` outputs a string starting with `#`, which interferes with the targeting.